os/services/shell/shell-commands.c

```diff
@@ -862,7 +888,9 @@ const struct shell_command_t builtin_shell_commands[] = {
   { "rpl-status",           cmd_rpl_status,           "'> rpl-status': Shows a summary of the current RPL state" },
   { "rpl-nbr",              cmd_rpl_nbr,              "'> rpl-nbr': Shows the RPL neighbor table" },
 #endif /* ROUTING_CONF_RPL_LITE */
+#if NETSTACK_CONF_WITH_IPV6
   { "routes",               cmd_routes,               "'> routes': Shows the route entries" },
```

Comment:
Looks good overall. Should we move this to the `#if NETSTACK_CONF_WITH_IPV6`  block above so we can have them all in the same place?

Reply:
agree, will do!

Reply:
done